src/gstvimbasrc.c

```diff
@@ -377,21 +377,21 @@ static void gst_vimbasrc_class_init(GstVimbaSrcClass *klass)
         g_param_spec_int(
             "offsetx",
             "OffsetX feature setting",
-            "Horizontal offset from the origin to the region of interest (in pixels).",
+            "Horizontal offset from the origin to the region of interest (in pixels). If no explicit value is passed the RoI will be centered in the sensor.",
             0,
             G_MAXINT,
-            0,
+            G_MAXINT,
             G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
     g_object_class_install_property(
         gobject_class,
         PROP_OFFSETY,
         g_param_spec_int(
             "offsety",
             "OffsetY feature setting",
-            "Vertical offset from the origin to the region of interest (in pixels).",
+            "Vertical offset from the origin to the region of interest (in pixels). If no explicit value is passed the RoI will be centered in the sensor.",
```

Comment:
Same as above for `offsetx`

```suggestion
            "Vertical offset from the origin to the region of interest (in pixels). If -1 is passed the ROI will be centered in the sensor along the vertical axis.",
```